.github/workflows/docker-build.yaml

```diff
@@ -465,6 +477,73 @@ jobs:
             -   name: Cypress container logs
                 if: ${{ failure() }}
                 run: docker compose logs cypress
+    tests-storefront-base:
```

Comment:
I would name it so it's clear that this step has to do something with cypress/acceptance tests.
for example `tests-storefront-acceptance-regenerate`

Reply:
Fixed